src/pages/ReimbursementAccount/PlaidOAuthPage.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import ONYXKEYS from '../../ONYXKEYS';
+import compose from '../../libs/compose';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import Navigation from '../../libs/Navigation/Navigation';
+import {
+    addPersonalBankAccount, setupWithdrawalAccount,
+} from '../../libs/actions/BankAccounts';
+import AddPlaidBankAccount from '../../components/AddPlaidBankAccount';
+
+const propTypes = {
+    /** Plaid SDK token to use to initialize the widget */
+    plaidLinkToken: PropTypes.string,
+
+    ...withLocalizePropTypes,
+};
+
+const PlaidOAuthPage = ({
+    plaidLinkToken,
+    translate,
+}) => {
+    const receivedRedirectSearchParams = (new URL(window.location.href)).searchParams;
```

Comment:
Unsure if this is possible, but can we get these params from react navigation? I thought maybe query params will just show up in route props, but not sure.

Reply:
we can